### ISSUE TITLE
Sending NDI example?

### ISSUE THREAD

**Issue by cliqer** [Streampunk/grandiose#14]
Using @jensstigaard's [repo](https://github.com/jensstigaard/grandiose) works like a charm on receiving streams but how does `sendVideo` works?

Tried with `grandiose.sendVideo()` and `send()` and I do receive the test patterns (even though the whole app freezes) but couldn't figure out how to use MediaStream Capture as a source.

**Comment by justweb1** [comment]
sending was not implemented yet and the maintainer has not updated this repo in 2 years.

**Comment by cliqer** [comment]
@jensstigaard has added sending NDI and seems to work just fine.
Can't understand how it works and while 'issues' are not available on a forked repo had to request this here :)

**Comment by jensstigaard** [comment]
Perhaps take a look at this promising fork
https://github.com/ianshade/grandiose/tree/feat/sending

**Comment by numenorien0** [comment]
Yes, but there's no explanation or example. Do you have an example ?

**Comment by jensstigaard** [comment]
No not really. But perhaps take a look at the great contribution from [rse](https://github.com/rse/grandiose).
There is [an example using .send here]( https://github.com/rse/vingester/blob/master/vingester-browser-worker.js)

**Comment by billygerhard** [comment]
@jensstigaard Your fork seems the most up to date, are you going to pull in that request from rse to yours?
